build.sbt

```diff
@@ -13,8 +10,8 @@ lazy val publishLocalSignedIfRelevant =
   taskKey[Unit]("Runs publishLocalSigned but only if scalaVersion in crossScalaVersions")
 Global / publishLocalSignedIfRelevant := PgpKeys.publishLocalSigned.value
 
-organization in ThisBuild := "org.typelevel"
-scalafixDependencies in ThisBuild += "org.typelevel" %% "simulacrum-scalafix" % "0.5.3"
+(ThisBuild / organization) := "org.typelevel"
```

Comment:
I think all those parens are unneeded, right?

Reply:
Hmm good point, I just ran the scalafix rule - somehow I didn't notice the parens